src/SSVOpenHexagon/Components/CPlayer.cpp

```diff
@@ -19,35 +19,29 @@
 #include <SFML/System/Vector2.hpp>
 #include <SFML/Graphics/Color.hpp>
 
-namespace hg
-{
+namespace hg {
```

Comment:
For the future, avoid changing the formatting of code. It makes the diffs bigger than they need to be. I would recommend using `clang-format` using the `.clang-format` style available in the repository root.

Reply:
It's my IDE that does this, didn't found a function to disable this auto formatting unfortunately.